.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+<!--
+Thanks for sending a pull request!
+
+In case you are changing only tests for a specific environment and don't need to
+run all test environments, add the following line to your PR description on a
+separate line! [CI:TOXENVS] foo,bar
+
```

Comment:
maybe mention that `build,all,repository,metadata,multistage` is always executed in addition to what is specified

Reply:
I noticed that in some of my PRs, GitHub interface grabs the last commit message and it would possibly hide this template.

Will this template show all the time? or can GitHub hide it in some scenarios?